OpenSim/Simulation/Manager/Manager.cpp

```diff
@@ -870,6 +882,18 @@ void Manager::initialize(SimTK::State& s, double dt )
 }
 //_____________________________________________________________________________
 /**
+* set and initialize a SimTK::TimeStepper
+*/
+void Manager::initializeTimeStepper(
+    const SimTK::System& sys, const SimTK::State& state)
```

Comment:
I totally agree. It does look like a search for `setSystem()` shows that this is the only use case. The question is whether or not this change belongs in this PR. I'm happy to add it on since (in theory) it should be easy to remove that functionality from `Manager`.